Code/Tools/ProjectManager/Source/CreateAGemScreen.h

```diff
@@ -92,12 +114,24 @@ namespace O3DE::ProjectManager
         QRadioButton* m_gemDetailsTab = nullptr;
         QRadioButton* m_gemCreatorDetailsTab = nullptr;
 
-        GemInfo m_createGemInfo;
+        GemInfo m_focalGemInfo;
```

Comment:
focal? as in the focused gem? maybe it'd be simple enough to just call it `m_gemInfo` and not characterize what kind of gem info it is in this class?

Reply:
I call it `m_focalGemInfo` because the `EditGem` class actually retains the `m_oldGemInfo`  temporarily when pre-filling values and supplying necessary arguments to the python edit gems function.

Reply:
It looks like you just need to save the old gem name - perhaps you could use `m_gemInfo` to store the final info and `m_originalName` to store the name needed to update the gem info?

Reply:
I'll opt for `m_oldGemName` for further clarity